mesheryctl/pkg/utils/platform.go

```diff
@@ -225,6 +226,110 @@ func GetChannelAndVersion(currCtx *(config.Context)) (string, string, error) {
 	return channel, version, nil
 }
 
+func GetDeploymentVersion(filePath, fileName string) (string, error) {
```

Comment:
I do not see a reason for passing fileName argument, except for logging, can this be hardcoded in that case?

Reply:
agreed @hexxdump 
Removed it now.